.goreleaser.yaml

```
@@ -0,0 +1,63 @@
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    main: ./cmd/lava
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - "7"
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
```

The windows ones can go away.

```suggestion
    ignore:
      - goos: darwin
        goarch: arm
```